go.mod

```diff
@@ -33,5 +33,3 @@ require (
 replace github.com/metal3-io/cluster-api-provider-metal3/api => ./api
 
 replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.4.4
-
-require github.com/opencontainers/image-spec v1.0.2 // indirect
```

Comment:
This is not needed anymore? We still import it in the [main](https://github.com/metal3-io/cluster-api-provider-metal3/blob/f88d0a9896c5f8734648a9d83e25c4096fabf34a/go.mod#L78) branch AFAIK

Reply:
good catch. This is wrongly removed

Reply:
@furkatgofurov7 I have double checked it. `go mod tidy ` is removing it for `release-0.5` branch but not for `main `branch. So I am leaving it as it is.

Reply:
@kashifest agree, checked it locally, seems we do not need this package anymore.